### DISCUSSION TITLE
Using Ring Alarm Unmonitored in HA: Flexible Setup Advice?

### DISCUSSION THREAD

**Discussion by apzuckerman** [tsightler/ring-mqtt#944] [Support Q&A]
I'm planning to move away from SimpliSafe and Arlo in favor of non-subscription-based cameras and a video doorbell. To complement this switch, I’m considering using Ring’s alarm system, including their window and door sensors and alarm pad.

I’d like to start without professional monitoring but want the option to enable monitoring later if needed. Does anyone know if this setup will work unmonitored and what limitations I might face without a subscription? Also, how well does this integrate with Ring-MQTT?

Thanks for the help!

**Comment by tsightler** [comment]
I don't even attempt to keep up with what Ring features require subscriptions vs do not as, quite honestly, I think most Ring devices are nearly useless without the subscription, so if you don't like subscriptions, best bet is to not purchase Ring devices.  Note that I don't test ring-mqtt without a subscription, although it should theoretically work fine for most cases.

However, the simple answer is, if you can do it in the app, then ring-mqtt can also do it, so your best bet is to determine what the app allows without a subscription.  Note that, subscription or not, Ring is a cloud based product, and they can change what they allow/disallow without a subscription at any time, although, historically, they've not retroactively changed behavior.  For example, new alarm purchases require a subscription for remote arming/disarming, however, if you purchased your alarm before the cutoff date of that capability becoming pay only, then it's still possible to arm/disarm without a subscription.

It's very important to understand what that means, remote arming/disarming means that arming/disarming via the Ring app isn't possible, which also means ring-mqtt can't arm/disarm without a subscription if you purchased after the cutoff date.  Of course, since the devices are still cloud powered in all cases, Ring is free to change these terms at an time, they could completely remove arming/disarming next month for all I know, or completely block all free use like MyQ did some time ago, although I've see zero signs of this.

Personally, I would not purchase a Ring product if I did not expect to keep a subscription for it.  For me, Ring Alarm has easily been worth it as I've been on the $100/year plan since it's introduction, having the extra automation features via ring-mqtt is just a bonus and was an unbelievable value compared to what I paid before for a professionally monitored alarm with cameras.  However, as Ring forces us legacy customers to these new plans, the value prop is going way, way down, although still pretty good vs competition, so I get it from a business perspective, but I'm not convinced I will stay with it at this point.   My last annual renewal on the legacy plan was in Nov, so I'll guess I'll have to decide come Nov 2025.

**Comment by apzuckerman** [comment]
That's helpful context, thanks. Any idea what features I would get if I'm
subscribed but not using the doorbell or ring cameras? If you had to start
from scratch today, what would you do? Appreciate the insightful response.
You clearly took some time to respond and I appreciate it.

On Wed, Dec 4, 2024 at 2:42 PM tsightler ***@***.***> wrote:

> I don't even attempt to keep up with what Ring features require
> subscriptions vs do not as, quite honestly, I think most Ring devices are
> nearly useless without the subscription, so if you don't like
> subscriptions, best bet is to not purchase Ring devices. Note that I don't
> test ring-mqtt without a subscription, although it should theoretically
> work fine for most cases.
>
> However, the simple answer is, if you can do it in the app, then ring-mqtt
> can also do it, so your best bet is to determine what the app allows
> without a subscription. Note that, subscription or not, Ring is a cloud
> based product, and they can change what they allow/disallow without a
> subscription at any time, although, historically, they've not retroactively
> changed behavior. For example, new alarm purchases require a subscription
> for remote arming/disarming, however, if you purchased your alarm before
> the cutoff date of that feature, then it's still possible to arm/disarm.
>
> It's very important to understand what that means, remote arming/disarming
> means that arming/disarming via the Ring app, which also means ring-mqtt
> can't arm/disarm without a subscription if you purchased after the cutoff
> date. Of course, since the devices are still cloud powered in all cases,
> Ring is free to change these terms at an time, the could completely remove
> arming/disarming next month for all I know, or completely block all free
> use like MyQ did some time ago.
>
> Personally, I would not purchase a Ring product if I did not expect to
> keep a subscription for it. For me, Ring Alarm has easily been worth it as
> I've been on the $100/year plan since it's introduction, having the extra
> automation features via ring-mqtt is just a bonus and was an unbelievable
> value compared to what I paid before for a professionally monitored alarm
> with cameras. However, as Ring forces us legacy customers to these new
> plans, the value prop is going way, way down, still good vs competition, so
> I get it from a business perspective, but I'm not convinced I will stay
> with it at this point.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tsightler/ring-mqtt/discussions/944#discussioncomment-11465288>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIJMZXAZFZAOU7QQHVVS5T2D5LKDAVCNFSM6AAAAABTA35VE6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBWGUZDQOA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Adam Zuckerman
Eisenhower Fellow
LinkedIn <https://www.linkedin.com/in/adamzuckerman/> | 202.270.9085

**Reply by tsightler** [reply]
I think the biggest features for Ring Alarm are cellular backup (although ring-mqtt functionality is quite limited when alarm operating on cellular backup, but at least app alerts should still work) and remote arming/disarming.  That's what $100/year gets you.  For the additional $10/mo, you get professional monitoring, and for another $100/year, you can get access to panic/SOS services.

But, I'm not a Ring salesperson and again, I don't really follow it that closely, so I would not rely 100% on my advice.

Starting from scratch, I would use local cameras (as you are planning to do) and use Ring Alarm only if I definitely wanted professional monitoring.  If I didn't care about professional monitoring, I would just use Home Assistant and something like Alarmo.  Maybe I'd still buy Ring sensors (they are just Z-wave sensors), so, in the future, if I decided I wanted professional monitoring, I could just buy the Ring Alarm base and re-pair the sensors to it.

If you don't care about remote arming/disarming or cellular backup, maybe Ring Alarm without a subscription is OK, but I could not live without being able to automate arming/disarming, it's quite literally why I created ring-mqtt in the very first place as my ancient alarm system prior to Ring had scheduling capability and I could not believe Ring didn't have this (I think they do now, but I believe it also requires a subscripting as it counts as "remote" arming/disarming, although I'm not sure of this).

**Comment by iamenigmatic** [comment]
Non-subscriber here.  Everything works perfectly without monitoring.  I have tested 1st and 2nd gen sensors for door and motion and have also integrated (but removed) first alert smoke/co alarms.  @tsightler is beyond amazing in keeping this working perfectly.  I run Alarmo on top of the ring-mqtt which adds extra flexibility to what you can do.  You should make the switch.

EDIT:  You can remotely arm and disarm without subscription.  There are no differences between self-monitor and paid.

**Reply by tsightler** [reply]
Just to be clear, did you purchase before or after the March 29th, 2023 cutoff data for remote arming/disarming.  I know that digital arming/disarming requires a subscription for users that purchased after this date.  That is, by far, the biggest drawback, and some users have even come to this forum to complain that somehow I was responsible for them purchasing a Ring alarm that they couldn't use because the docs for this project didn't state that, like somehow I made this project so I'm now responsible for keeping up with every policy change Ring makes.

**Reply by iamenigmatic** [reply]
Anybody that uses this add-on should not be holding you responsible for anything.  You have built an integration that works and the use is by choice for each person.  That said, yes, I have been a ring user for a long time.  Everybody actually had their ability via Ring app to arm/disarm removed.  I do still have an old apk on my device but this integration receives data from the source, so that isn't changing anything.  In my opinion, the best way for @apzuckerman to test this would be to purchase a ring system from costco.  Set it up and integrate.  If the arm/disarm is fully gone, return the system to costco and no money is lost.  I honestly can't say if a brand new account would be limited in this ability, but I also can't see why they would tell me, a long time user, that the arm/disarm would be removed but it still works for me via your add-on.  I fully believe the costco purchase and test is worth it.  I guess at the very least it would stand as a final test of what is fully available to a new user.  I can only confirm that literally everything works for me.  I can even setup custom scripts to recreate a panic scenario that will call 911 for me on my phone using Automate app for android.

BTW: @tsightler I really do think you are amazing for keeping this going.  Your work and the work of others on this is beyond appreciated.  I searched on how to donate to you and saw one of your old posts about passing on the donation to source projects for this project.  You are awesome!

**Reply by tsightler** [reply]
The actual statement from Ring specifically said that the feature would continue to be available if you "owned an Alarm before this date".   Here's a screenshot of what was sent at the time:

![image](https://github.com/user-attachments/assets/35d4fea1-7470-4f36-afef-91185b9aa91f)

Based on this, you should still be able to arm/disarm from the app.  However, I already know for a fact that this API is blocked for users that have purchased after this date, I've had more than enough come here to complain about it, like somehow I could do something about it.  It's almost like people think I work for Ring or something, when, in reality, I have nothing to do with them other than owning their products.

I actually think Ring Alarm will still be a pretty great value, I mean, even after next year it will only be $200/yr, which is still an amazing value for a professionally monitored alarm, I remember when I used to pay $50/month to ADT for similar service.  It's just that it's difficult to swallow a 100% price increase even though, to be fair to Ring, it will have been 7 years at the old $100/year rate, so that is pretty good.

**Reply by iamenigmatic** [reply]
I do feel lucky to be on that grandfathered list, I guess.  That sucks for new users.  If were me and a new user, I would still go with the ring system and integration as subscriber.  Agreed, the price increase is classic late stage capitalism, but I have been around alarm systems for a very long time, and this fee is still reasonable compared to other services.  Adding in your add-on brings in extra value to the user that can't be replaced.  In the case of this particular discussion, I would still make this switch from Arlo to Ring.
